statviz/src/views/Statistics/StatisticsView.tsx

```diff
@@ -0,0 +1,45 @@
+import { gql, useQuery } from "@apollo/client";
+import { BeneficiaryDemographicsQuery, BeneficiaryDemographicsQueryVariables, HumanGender } from "../../types/generated/graphql";
+import DemographicChart, { IDemographicCube, IDemographicFact } from "./components/DemographicChart";
+
+const DEMOGRAPHIC_QUERY = gql`
+  query BeneficiaryDemographics {
+    beneficiaryDemographics {
+      facts {
+        count
+				createdOn
+				age
+				gender
+      }
+      dimensions {
+        tag {
+          __typename
+          name
+          id
+        }
+      }
+    }
+  }
+`
+
+export default function StatisticsView() {
+
+  const { data, loading, error } = useQuery<
+    BeneficiaryDemographicsQuery,
+    BeneficiaryDemographicsQueryVariables
+  >(DEMOGRAPHIC_QUERY);
+
+  if (loading || data?.beneficiaryDemographics === undefined) {
```

Comment:
The second part of the if does not make much sense. If an error happens, data?.beneficiaryDemographics could be undefined and we would show the loading state.

ACTION ITEM: add error handling.